integration_tests/fixtures/forseti/scripts/wait-for-forseti.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Block until the Forseti startup script has finished running.
+
+MAX_WAIT=900
+
+echo "Waiting for up to $MAX_WAIT seconds for Forseti to be ready."
+
+for _ in $(seq 1 $MAX_WAIT); do
+  if [[ -f /etc/profile.d/forseti_environment.sh ]]; then
+    echo "Forseti is not ready yet"
```

Comment:
Just to double check, what is the state here?  Forseti is not ready?  Then why `exit 0`?

Reply:
I have removed that `echo` command that said Forseti is not ready yet. Essentially, Forseti is ready 60 seconds after the condition in the `if` block is met. Updated the script.